utils/swift_build_support/swift_build_support/products/wasmkit.py

```diff
@@ -73,11 +73,10 @@ def cli_file_path(cls, build_dir):
 
 
 def run_swift_build(host_target, product, swiftpm_package_product_name, set_installation_rpath=False):
-    if product.args.build_runtime_with_host_compiler:
-      swift_build = product.toolchain.swift_build
-    else:
-      # Building with the freshly-built SwiftPM
-      swift_build = os.path.join(product.install_toolchain_path(host_target), "bin", "swift-build")
+    swift_build = os.path.join(product.install_toolchain_path(host_target), "bin", "swift-build")
+
+    if not os.path.exists(swift_build) or product.args.build_runtime_with_host_compiler:
+        swift_build = product.toolchain.swift_build
```

Comment:
Can you print a message just in case to indicate that the artifact built with the host toolchain can't be shipped?

Reply:
Would this be suitable?

```suggestion
        print("WARNING: running local development code path, don't use for deployment!")
        swift_build = product.toolchain.swift_build
```